ArduPlane/quadplane.h

```diff
@@ -505,6 +505,7 @@ class QuadPlane
         OPTION_ALLOW_FW_LAND=(1<<2),
         OPTION_RESPECT_TAKEOFF_FRAME=(1<<3),
         OPTION_MISSION_LAND_FW_APPROACH=(1<<4),
+	OPTION_FS_ACTION=(1<<5),
```

Comment:
Whitespace, and should be named `OPTION_FS_QRTL` or something to indicate what it does rather then just `FS_ACTION`

Reply:
done